system/lib/emmalloc.c

```diff
@@ -254,9 +257,10 @@ static int compute_free_list_bucket(size_t allocSize) {
 #define DECODE_CEILING_SIZE(size) ((size_t)((size) & ~FREE_REGION_FLAG))
 
 static Region *prev_region(Region *region) {
-  size_t prevRegionSize = ((size_t*)region)[-1];
+  size_t * siz = (size_t*)region;
```

Comment:
This would be closer to the conventions elsewhere in this file:
```suggestion
  size_t *size = (size_t*)region;
```